### PR TITLE
TW-1055: Add Temple Token to Market page

### PR DESCRIPTION
**Pull request by lendihop**
https://madfish.atlassian.net/jira/software/projects/TW/boards/4?selectedIssue=TW-1055

### CODE REVIEW

**Review by herkoss — APPROVED** [review]
GJ!

File: ios/Podfile.lock
Comment (herkoss) [inline ios/Podfile.lock:1]: Is it okay that file has been changed?

**Review by herkoss — APPROVED** [review]
GJ!

File: ios/Podfile.lock
Comment (lendihop) [reply ios/Podfile.lock:1]: the problem is that ios build fails without this change

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_929

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_929

**Pull request merged**